packages/service-library/src/servicelib/client_session.py

```diff
@@ -22,8 +23,10 @@ def get_client_session(app: web.Application) -> ClientSession:
         # it is important to have fast connection handshakes
         # also requests should be as fast as possible
         # some services are not that fast to  reply
-        # Setting the time of a request using this client session to 5 seconds total
-        timeout_settings = ClientTimeout(total=20, connect=1, sock_connect=1)
+        # Setting the time of a request using this client session to 5 seconds totals
+        timeout_settings = ClientTimeout(
+            total=get_http_client_request_total_timeout(), connect=5, sock_connect=5
```

Comment:
you changed sock_connect for some reason here? what is this for? websocket connection?
should they also have an assigned variable? if not they should not be there with some number. or if this is the basis for all the others create a variable with a meaningful name

Reply:
The sued arguments (from [the docs](https://docs.aiohttp.org/en/latest/client_quickstart.html#timeouts)):
- **connect** The maximal number of seconds for connection establishment of a new connection or for waiting for a free connection from a pool if pool connection limits are exceeded.
- **sock_connect** The maximal number of seconds for connecting to a peer for a new connection, not given from a pool.

Because we are using service to service communication my Idea is to keep this as low as possible (initially 1 second all over the place). You fail fast if the remote service is not responding. I've raised them to 5 seconds to allow for slow hardware as discussed before.